complete/_rg

```diff
@@ -44,6 +44,10 @@ _rg() {
     '(: * -)'{-h,--help}'[display help information]'
     '(: * -)'{-V,--version}'[display version information]'
 
+    + '(hybrid)' # hybrid regex options
+    '--auto-hybrid-regex[dynamically use PCRE2 if necessary]'
```

Comment:
Kind of a nit-pick, but describing the affected functionality differently between the enable and disable options feels a little odd (and inconsistent with other options)